frontend/src/features/admin-form/settings/SettingsPage.tsx

```diff
@@ -48,14 +40,32 @@ export const SettingsPage = (): JSX.Element => {
 
   const { ref, onMouseDown } = useDraggable<HTMLDivElement>()
 
-  const [tabIndex, setTabIndex] = useState(
-    settingsTabsOrder.indexOf(settingsTab ?? ''),
+  // Note: Admins are not redirected to /general on invalid settings tabs as we
+  // don't want to do this prematurely before displayPayments can be determined.
+  const tabConfig = useMemo(() => {
+    const baseTabs = [
```

Comment:
Could the whole `tabConfig` be moved to a top-level constant instead of `<SettingsPage />`? 

1. nothing in `<SettingsPage />` would affect the contents of `tabConfig`
2. It regenerates a new baseTabs whenever `<SettingsPage />` unmounts (not exactly a significant perf issue)

Reply:
i.e., 

```
const tabConfig = [
  { label: 'General', icon: BiCog, component: SettingsGeneralPage },
  { label: 'Singpass', icon: BiKey, component: SettingsAuthPage },
  {
    label: 'Twilio credentials',
    icon: BiMessage,
    component: SettingsTwilioPage,
  },
  { label: 'Webhooks', icon: BiCodeBlock, component: SettingsWebhooksPage },
  { label: 'Payments', icon: BiDollar, component: SettingsPaymentsPage },
]
```

Reply:
Ahh that's a good point. Previously, the payments (now always visible) and the email notifications (removed) tabs were added dynamically, which I also checked for changes in `displayEmailNotif...`. Would keeping it as a top level constant also be a better choice when we have dynamically shown tabs in the future?